docusaurus/src/components/RequestERD/EmailModal.module.css

```diff
@@ -0,0 +1,77 @@
+.modalOverlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: var(--modal-overlay-background);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 1000;
+}
+
+.modalContentHeader {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+.modalContentHeader p {
+  margin-bottom: 0;
+  font-size: 0.8em;
+}
+
+.modalContentClose {
+  background: none;
+  border: none;
+  cursor: pointer;
+  margin-left: auto;
+  margin-right: 0;
+}
+.modalContent {
+  background: var(--color-white);
+  padding: 20px;
+  border-radius: 5px;
+  max-width: 400px;
+  width: 100%;
+  font-size: 0.8em;
+}
+
+.modalContentForm {
+  display: flex;
+  width: 100%;
+  gap: 8px;
+
+}
+.modalContentInputContainer {
+  flex-grow: 1;
+}
+.modalContentInput {
+  border: 1px solid var(--color-grey-400);
+  border-radius: 6px;
+  padding: 10px;
+  width: 100%;
+}
+.modalContentInputError {
+  border: 1px solid var(--color-red-400);
+}
+.inputErrorMessage {
+  color: var(--color-red-400);
+  font-size: 0.8em;
+  margin-bottom: 0;
+}
+
+.modalContentButton {
+  background-color: var(--ifm-color-primary);
+  color: white;
+  border: none;
+  padding: 10px 20px;
+  border-radius: 6px;
+  cursor: pointer;
+  max-height: 38px;
+}
+
+.modalContentButton:disabled {
+  background-color: var(--ifm-color-primary-light);
```

Comment:
In the airbyte webapp, we usually just set the opacity to 50% for disabled buttons - I think this current color looks a little bit too close to the non-disabled color, so it wasn't as clear to me that it was disabled. 

Setting opacity to 50% instead would probably make this distinction clearer